java/.ci/Jenkinsfile

```diff
@@ -333,6 +333,32 @@ EOF
                         }
                     }
                 }
+                stage('S3 TO BigQuery (avro)'){
+                    steps {
+                        retry(count: stageRetryCount) {
+                        withCredentials([usernamePassword(credentialsId: 'aws-s3-ro-credentials',
+                        passwordVariable: 'S3_SECRET', usernameVariable: 'S3_KEY')]) {
+                            sh '''
+                                export SKIP_BUILD=true
+
+                                cd java
+
+                                bin/start.sh \
+                                -- --template S3TOBIGQUERY \
+                                --templateProperty project.id=yadavaja-sandbox \
```

Comment:
Very minor change. We usually use `$GCP_PROJECT` instead of hard coding project id. Otherwise all good.